python/nav/portadmin/handlers.py

```diff
@@ -25,6 +26,17 @@
 _logger = logging.getLogger(__name__)
 
 
+@dataclass
+class PoeState:
+    """Class for defining PoE states.
+    `state` is the value used on the device itself.
+    `name` is a human readable name for the state
+    """
+
+    state: Union[str, int]
+    name: str
+
+
```

Comment:
For cisco equipment having a name and internal state value makes sense since you can use SNMP for doing all this, where the MIB maps value 1 to DISABLED, 2 to ENABLED etc, but for juniper you basically just run commands for either enabling or disabling, so in that case the `state` value has no purpose. Might make sense then to make `state` optional. But it is useful for cisco, and perhaps other vendors where u can use pure SNMP.